### DISCUSSION TITLE
PrismaLayer is missing rhel-openssl engine

### DISCUSSION THREAD

**Discussion by sandreev** [jetbridge/sst-prisma#54] [Q&A] [answered]
The output of installing `@prisma/client` and `@prisma/engines` doesn't include the `rhel-openssl` engine and as a result, the `PRISMA_QUERY_ENGINE_LIBRARY` points to a non-existing file. I double-checked the layer that SST creates and it is missing the rhel-openssl engine. Can you help to resolve this issue?

**Comment by revmischa** [comment]
Try adding the rhel-openssl target here https://github.com/jetbridge/sst-prisma/commit/a82ed9eb3d91b462a99ef55698b88c6ad452ebe4#diff-93ead908b5ffa6f6e995757b3739fb02b4e9f8d0ea1da0ec34a9c03c1a33f9bfR7 in schema.prisma

**Reply by sandreev** [reply]
Thank you for the response @revmischa! However, the way Prisma layer is being created, it doesn't take any files right now from the repo - it installs the @prisma/client and @prisma/engines; removes the unneeded engines and finishes the layer. I don't really see any logic right now that would copy the rhel engine to the layer in the existing codebase. Unless, I am missing something obvious...

**Reply by sandreev** [reply]
I found a way to download the rhel engine from the Prisma mirror within the docker container that is used to create the layer but I was hoping that I missed something straight-obvious in your example.

**Reply by sandreev** [reply, marked as answer]
Ok, I change the asset-input from "." to "node_modules/.prisma/client and added a command in layer bundling to mv the rhel-engine from asset-input to engineDir. This fixed the issue
